snap/squashfs/squashfs.go

```diff
@@ -72,10 +60,10 @@ func (s *Snap) MetaMember(metaMember string) ([]byte, error) {
 }
 
 // Install just copies the blob into place (unless it is used in the tests)
-func (s *Snap) Install(instDir string) error {
+func (s *Snap) Install(targetPath, instDir string) error {
```

Comment:
Maybe `mountDir` instead of `instDir` for the parameter here to make it more clear what the difference is?